br/pkg/lightning/importer/import.go

```diff
@@ -2026,6 +2027,11 @@ func (rc *Controller) enforceDiskQuota(ctx context.Context) {
 			for _, engine := range largeEngines {
 				// Use a larger split region size to avoid split the same region by many times.
 				if err := localBackend.UnsafeImportAndReset(ctx, engine, int64(config.SplitRegionSize)*int64(config.MaxSplitRegionSizeRatio), int64(config.SplitRegionKeys)*int64(config.MaxSplitRegionSizeRatio)); err != nil {
+					engineIns := localBackend.EngineMgr.RLockEngine(engine)
```

Comment:
Sorry I just noticed that this error will not be reported to user, so we don't need to convert it. Please revert https://github.com/pingcap/tidb/pull/51277/commits/3fda42b6a98d6d475402b0b468294ecf59b9c118 except for the DupeResAlgErr change in pkg/ddl/ingest/backend.go